dreambooth/entrypoints/train.py

```diff
@@ -23,21 +24,23 @@
 
 def main():
     config = get_config(TrainConfig)
+    # Check if wandb env var is configured, if not, disable logging
+    use_wandb = os.environ.get("WANDB_API_KEY") is not None
 
     # --- Accelerator and logging setup ---
     config.logging_dir.parent.mkdir(parents=True, exist_ok=True)
     accelerator = Accelerator(
         gradient_accumulation_steps=config.gradient_accumulation_steps,
         mixed_precision=config.mixed_precision,
         project_dir=config.logging_dir,
-        log_with="wandb",
+        log_with="wandb" if use_wandb else None,

```

Comment:
Can we still log to console?

Reply:
Not directly with the Accelerator, but we are logging with the normal logger. They also have their own logger, but I don't think it would change much in this case.